src/shared/sam/debugger/pythonSamDebug.ts

```diff
@@ -138,9 +138,14 @@ export async function makePythonDebugConfig(
             // --ikpdb-protocol=vscode:
             //           For https://github.com/cmorisse/vscode-ikp3db
             //           Requires ikp3db 1.5 (unreleased): https://github.com/cmorisse/ikp3db/pull/12
-            config.debugArgs = [
-                `-m ikp3db --ikpdb-address=0.0.0.0 --ikpdb-port=${config.debugPort} -ik_ccwd=${ccwd} -ik_cwd=/var/task ${logArg}`,
-            ]
+            const debugArgs = `-m ikp3db --ikpdb-address=0.0.0.0 --ikpdb-port=${config.debugPort} -ik_ccwd=${ccwd} -ik_cwd=/var/task ${logArg}`
+
+            if (isImageLambda) {
+                const params = getPythonExeAndBootstrap(config.runtime)
+                config.debugArgs = [`${params.python} ${debugArgs} ${params.bootstrap}`]
+            } else {
+                config.debugArgs = [debugArgs]
```

Comment:
Did the sam expectations change? Or does this fix https://github.com/aws/aws-toolkit-vscode/issues/1442

Reply:
Fixes #1442

Reply:
I should add that the original error shown in the related issue is due `ikp3db` being built from source in the image. But it can be fixed by adding this to the Dockerfile:
```
RUN yum groupinstall -y "Development Tools"
```

This PR fixes the problem you would run into _after_ updating the Dockerfile.